cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -325,13 +322,16 @@ func del(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 			args.ContainerID, err)
 
 		// When IPAMD is unreachable, try to teardown pod network using previous result. This action prevents rules from leaking while IPAMD is unreachable.
-		// Note that connection error is still returned to kubelet so that delete retries until IPAMD can be notified.
+		// Note that no error is returned to kubelet as there is no guarantee that kubelet will retry delete, and returning an error would prevent container runtime
+		// from cleaning up resources. When IPAMD again becomes responsive, it is responsible for reclaiming IP.
 		if teardownPodNetworkWithPrevResult(driverClient, conf, k8sArgs, args.IfName, log) {
 			log.Infof("Handled pod teardown using prevResult: ContainerID(%s) Netns(%s) IfName(%s) PodNamespace(%s) PodName(%s)",
 				args.ContainerID, args.Netns, args.IfName, string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_NAME))
+		} else {
+			log.Infof("Could not teardown pod using prevResult: ContainerID(%s) Netns(%s) IfName(%s) PodNamespace(%s) PodName(%s)",
```

Comment:
this shall be a log.errorf or at least `warnf`

Reply:
The most common case where this happens is the pod already being deleted. Kubelet often sends the DEL request 2-3 times, even if the first one succeeded. I left this as "info" to be inline with the other info above, where IPAMD sees the container as "not found"